action-server/covidflow/actions/test_navigation_form.py

```diff
@@ -33,6 +36,8 @@
 END_FORM_SLOT = "test_navigation__end_form"
 
 CHILDREN_CLIENTELE_REGEXP = re.compile(r"no_children_under_(\d{1,2})(_months)?")
+LONG_TIME_FORMAT = {"en": "%-I:%M%p", "fr": "%-Hh%M"}
+SHORT_TIME_FORMAT = {"en": "%-I%p", "fr": "%-Hh"}
```

Comment:
Although, it also makes it less testable if in the domain files, let's say if the format changes and is invalid. We would have to test it in the dialogue tests (this could be left out for the moment).